tests/unit/models/task-test.js

```diff
@@ -17,6 +22,72 @@ test('it exists', function(assert) {
   assert.ok(!!model);
 });
 
+test('it should have all of it\'s attributes', function(assert) {
+  assert.expect(8);
+  
+  let task = this.subject();
+  let attributes = Object.keys(task.toJSON());
+
+  assert.ok(attributes.includes('body'), 'task should have the body attribute');
+  assert.ok(attributes.includes('insertedAt'), 'task should have the insertedAt attribute');
+  assert.ok(attributes.includes('likesCount'), 'task should have the likesCount attribute');
+  assert.ok(attributes.includes('markdown'), 'task should have the markdown attribute');
+  assert.ok(attributes.includes('number'), 'task should have the number attribute');
+  assert.ok(attributes.includes('taskType'), 'task should have the taskType attribute');
+  assert.ok(attributes.includes('status'), 'task should have the status attribute');
+  assert.ok(attributes.includes('title'), 'task should have the title attribute');
+});
+
+test('it should belong to a project', function(assert) {
+  assert.expect(2);
+
+  const task = this.store().modelFor('task');
+  const relationship = get(task, 'relationshipsByName').get('project');
+
+  assert.equal(relationship.key, 'project', 'has relationship with project');
+  assert.equal(relationship.kind, 'belongsTo', 'kind of relationship is belongsTo');
+});
+
+test('it should belong to a user', function(assert) {
+  assert.expect(2);
+
+  const task = this.store().modelFor('task');
+  const relationship = get(task, 'relationshipsByName').get('user');
+
+  assert.equal(relationship.key, 'user', 'has relationship with user');
+  assert.equal(relationship.kind, 'belongsTo', 'kind of relationship is belongsTo');
```

Comment:
These tests are starting to seem really duplicative. Wondering if we can instead do something like `assert.hasManyRelationship('task', 'user')` or `assert.belongsToRelationship('task', 'project')` which just wraps everything.